packages/rancher-istio/charts/README.md

```diff
@@ -1,40 +1,54 @@
 
-If you remove dependent CRD charts prior to removing rancher-istio, you may encounter the following error::
+**If you remove dependent CRD charts prior to removing rancher-istio, you may encounter the following error:**
+`Error: uninstallation completed with 1 error(s): unable to build kubernetes objects for delete: unable to recognize "": no matches for kind   "MonitoringDashboard" in version "monitoring.kiali.io/v1alpha1"`
 
-`Error: uninstallation completed with 1 error(s): unable to build kubernetes objects for delete: unable to recognize "": no matches for kind "MonitoringDashboard" in version "monitoring.kiali.io/v1alpha1"`
+## Addons
+The addons that are included with rancher-istio are:
 
-# Addons
+- Kiali
+- Jaeger
 
-## Kiali
+Each addon has additional customization and dependencies required for them to work as expected. Use the values.yaml to customize or to enable/disable each addon.
+### Kiali Addon
 
 Kiali allows you to view and manage your istio-based service mesh through an easy to use dashboard.
 
-####  Dependencies
-- rancher-monitoring chart or other Prometheus installation
+####  Kiali Dependencies
+##### rancher-monitoring chart or other Prometheus installation
 
 This dependecy installs the required CRDs for installing Kiali. Since Kiali is bundled in with Istio in this chart, if you do not have these dependencies installed, your Istio installation will fail. If you do not plan on using Kiali, set `kiali.enabled=false` when installing Istio for a succesful installation.
 
+####  Prometheus Configuration for Kiali
 > **Note:** The following configuration options assume you have installed the dependecies for Kiali. Please ensure you have Promtheus in your cluster before proceeding.
 
-The Monitoring app sets `prometheus.prometheusSpec.ignoreNamespaceSelectors=false` which means all namespaces will be scraped by Prometheus by default. This ensures you can view traffic, metrics and graphs for resources deployed in other namespaces.
+The Rancher Monitoring app sets `prometheus.prometheusSpec.ignoreNamespaceSelectors=false` which means all namespaces will be scraped by Prometheus by default. This ensures you can view traffic, metrics and graphs for resources deployed in other namespaces.
 
 To limit scraping to specific namespaces, set `prometheus.prometheusSpec.ignoreNamespaceSelectors=true` and add one of the following configurations to ensure you can continue to view traffic, metrics and graphs for your deployed resources. 
 
 1. Add a Service Monitor or Pod Monitor in the namespace with the targets you want to scrape.
 1. Add an additionalScrapeConfig to your rancher-monitoring instance to scrape all targets in all namespaces.
 
-####  External Services
+####  Kiali External Services
+
+See the app-readme.md for configuration options. 
```

Comment:
Is this true ? I don't see them in there, if its the docs link we should just link that here.

Reply:
this was left behind when I was moving stuff around - good catch - it has been removed in the latest commits.